### PR TITLE
Add and run flake8-isort in Travis builds

### PR DESCRIPTION
**Pull request by stephendonner**
Also not-yet **_fully_** ready for review, but I think this Should Just Work (tm).

### CODE REVIEW

**Comment by davehunt** [comment]
@stephendonner could you summarise the issues, and are you seeing them locally or just in CI?

**Comment by stephendonner** [comment]
Sure; https://travis-ci.org/citp/OpenWPM/jobs/407022965 shows the following two errors:

./test_simple_commands.py:10:1: I004 isort found an unexpected blank line in imports
./test_simple_commands.py:11:1: I001 isort found an import in the wrong position

But locally, I'm not seeing those reported, so I'm not sure which: (a) my clean flake8 run or (b) these two errors reported in Travis, are valid.

screen shot 2018-07-23 at 9 48 10 am

**Comment by davehunt** [comment]
As isort is trying to distinguish between the current Python project and third-party packages, it can become confused by what's installed in the current environment. One way around this is to [configure isort](https://github.com/timothycrosley/isort#configuring-isort) with known first-party and third-party modules, and to set the default section. Hopefully this will help you to get something running consistently between environments.

**Comment by stephendonner** [comment]
@englehardt thoughts/r?

File: test/openwpmtest.py [hunk 1]
```diff
@@ -62,6 +63,7 @@ def assert_py_pkg_installed(self, pkg):
         pkg_name_mapping = {"pyopenssl": "OpenSSL",
                             "beautifulsoup4": "bs4",
                             "python-dateutil": "dateutil",
+                            "flake8-isort": "flake8",
```
[the remark this answers is not in the document]
Comment (stephendonner) [reply test/openwpmtest.py]: Thanks; fixed in https://github.com/citp/OpenWPM/pull/208/commits/2e9b3775b0d4cb82d87228dfc234214a2dbeee6d hopefully.

**Pull request merged**